packages/shorebird_cli/lib/src/commands/collaborators/delete_collaborators_command.dart

```diff
@@ -0,0 +1,126 @@
+import 'dart:async';
+
+import 'package:collection/collection.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/auth_logger_mixin.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/shorebird_config_mixin.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// {@template delete_collaborators_command}
+/// `shorebird collaborators delete`
+/// Delete an existing collaborator from a Shorebird app.
+/// {@endtemplate}
+class DeleteCollaboratorsCommand extends ShorebirdCommand
+    with AuthLoggerMixin, ShorebirdConfigMixin {
+  /// {@macro delete_collaborators_command}
+  DeleteCollaboratorsCommand({
+    required super.logger,
+    super.buildCodePushClient,
+    super.auth,
+  }) {
+    argParser
+      ..addOption(
+        _appIdOption,
+        help: 'The app id that contains the collaborator to be deleted.',
+      )
+      ..addOption(
+        _collaboratorEmailOption,
+        help: 'The email of the collaborator to delete.',
+      );
+  }
+
+  static const String _appIdOption = 'app-id';
+  static const String _collaboratorEmailOption = 'email';
+
+  @override
+  String get description =>
+      'Delete an existing collaborator from a Shorebird app.';
+
+  @override
+  String get name => 'delete';
+
+  @override
+  Future<int>? run() async {
+    if (!auth.isAuthenticated) {
+      printNeedsAuthInstructions();
+      return ExitCode.noUser.code;
+    }
+
```

Comment:
This might be redundant, but should we add a check that shorebird is initialized here?

Reply:
It shouldn't be necessary since we prompt if a shorebird.yaml isn't detected.